packages/connect-known-chains/README.md

```diff
@@ -18,3 +18,9 @@ This package contains all the well known chain specifications. It is updated
 daily by a bot via pull request.
 
 You can find the chainspecs in the [specs](./specs/) directory.
+
+/*
+Aside: I wonder whether we should mvoe to pointing here for chainspecs:
+
+https://github.com/paritytech/chainspecs
```

Comment:
unless they package that repo up as an npm package, probably not imo. too many submodules to deal with

Reply:
Maybe we can get them to publish an npm package to CI or something automatically; I will have to ask!

Reply:
https://github.com/paritytech/chainspecs/issues/246